core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/EncodingFactory.java

```diff
@@ -66,6 +66,14 @@ public abstract ColumnPageEncoder createEncoder(TableSpec.ColumnSpec columnSpec,
    */
   public ColumnPageDecoder createDecoder(List<Encoding> encodings, List<ByteBuffer> encoderMetas,
       String compressor) throws IOException {
+    return createDecoder(encodings, encoderMetas, compressor, false);
+  }
+
+  /**
+   * Return new decoder based on encoder metadata read from file
```

Comment:
In the comment, can you describe what is the behavior when `fullVectorFill` is true?

Reply:
added comment